modules/core/src/lib/layer.ts

```diff
@@ -186,7 +186,6 @@ export type UpdateParameters<LayerT extends Layer> = {
 };
 
 type SharedLayerState = {
-  model?: Model;
```

Comment:
Changing to `[key:string]: unknown` would create a big cascade?

Reply:
Like what? This is not referenced by any layer. Many layers do not define a `state.model`.

Reply:
Then it should not be a problem to change. My point is to always prefer `unknown` over `any`. `any` is very deceptive in usage, especially in a mostly typed api. `unknown` requires the developer to be intentional.